thriftbp/example_client_test.go

```diff
@@ -4,9 +4,12 @@ import (
 	"context"
 
 	"github.com/apache/thrift/lib/go/thrift"
+	retry "github.com/avast/retry-go"
 	opentracing "github.com/opentracing/opentracing-go"
+
```

Comment:
nit: this is an exception in the style guide: https://github.com/reddit/baseplate.go/blob/master/Style.md#exceptions-1. we don't separate third party imports with baseplate.go imports in full file examples.